examples/add_person.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/golang/protobuf/proto"
```

Comment:
Separate out the standard library imports from the others. Also rename the protobuf import, and use it appropriately qualified below.

```go
import (
  "bufio"
  ...
  "strings"

  "github.com/golang/protobuf/proto"
  pb "github.com/google/protobuf/examples/tutorial"
)
```